base_layer/core/src/proof_of_work/monero_rx/mod.rs

```diff
@@ -741,20 +519,26 @@ mod test {
             output_mmr_size: 0,
             kernel_mr: vec![0],
             kernel_mmr_size: 0,
-            total_kernel_offset: RistrettoSecretKey::from(0),
-            total_script_offset: RistrettoSecretKey::from(0),
+            total_kernel_offset: Default::default(),
+            total_script_offset: Default::default(),
             nonce: 0,
             pow: ProofOfWork::default(),
         };
-        let monero_data = MoneroData::default();
-        let serialized = bincode::serialize(&monero_data).unwrap();
+        let monero_data = MoneroPowData {
```

Comment:
Is there not a default for this anymore?

Reply:
I'm 50-50 on this, but I didn't see it fitting well - what is a default Transaction? Does a default Merkle Proof make any sense? It was used in one place, so I went with rather not having the concept of Default MoneroPowData (which I can't clearly see as having any meaning)